packages/protobuf/src/next/to-json.ts

```diff
@@ -151,7 +155,25 @@ function reflectToJson(msg: ReflectMessage, opts: JsonWriteOptions): JsonValue {
       json[jsonName(f, opts)] = jsonValue;
     }
   }
-  // TODO: Add support for extensions.
+  const descSet = opts.descSet;
+  if (descSet?.getExtensionFor) {
```

Comment:
Shouldn't this be sufficient?

```suggestion
  if (descSet) {
```